opening-hours/src/localization/coordinates.rs

```diff
@@ -34,24 +29,23 @@ impl Coordinates {
             TimeEvent::Dusk => SolarEvent::Dusk(DawnType::Civil),
         };
 
-        let solar_day = SolarDay::new(self.lat, self.lon, date.year(), date.month(), date.day());
-        let timestamp = solar_day.event_time(solar_event);
-        chrono::DateTime::from_timestamp(timestamp, 0).expect("invalid timestamp")
+        let solar_day = SolarDay::new(self.0, date);
+        solar_day.event_time(solar_event)
     }
 
     /// Get latitude component.
     pub fn lat(&self) -> f64 {
-        self.lat
+        self.0.lat()
     }
 
     /// Get longitude component.
     pub fn lon(&self) -> f64 {
-        self.lon
+        self.0.lon()
     }
 }
 
 impl std::fmt::Display for Coordinates {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "({}, {})", self.lat, self.lon)
+        write!(f, "({}, {})", self.lat(), self.lon())
     }
 }
```

Comment:
I think we could just implement `Deref<Target = sunrise::Coordinates>` instead of re-implementing these helpers.